swagger/swagger-invocation/invocation-core/src/main/java/org/apache/servicecomb/swagger/invocation/exception/ExceptionToResponseConverters.java

```diff
@@ -42,8 +44,18 @@ public ExceptionToResponseConverters() {
   }
 
   public Response convertExceptionToResponse(SwaggerInvocation swaggerInvocation, Throwable e) {
-    ExceptionToResponseConverter<Throwable> converter =
-        exceptionToResponseConverters.getOrDefault(e.getClass(), defaultConverter);
+    ExceptionToResponseConverter<Throwable> converter = null;
```

Comment:
we can cache the result: Map<Class<?>, ExceptionToResponseConverter<Throwable> >
because the same exception, will always get the same converter

Reply:
It will not improve a lot of performance for doing that. Because class hierarchy will not be that high, 3 is most of the case. Caching too many keys may get worse performance. And error processing is not that performance critical.

Reply:
ok